lib/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicator.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxVulnerabilityIndicator.scss';
+
+export default function NxVulnerabilityIndicator({
+  criticalCount, severeCount, moderateCount, lowCount, layout, className, ...attrs }: Props) {
+
+  const layoutClasses = classnames('nx-vulnerability-indicator-container', className, {
+    [`nx-vulnerability-indicator-container--${layout}`]: layout,
+    ['nx-vulnerability-indicator-container--row']: !layout
+  });
+
+  return (
+    <div className={layoutClasses} {...attrs}>
```

Comment:
Would it make sense for this whole thing to be a `<dl>` with the count and text as `<dt>` and `<dd>`?

Reply:
> Would it make sense for this whole thing to be a `<dl>` with the count and text as `<dt>` and `<dd>`?

I see what you're thinking but It seems like a bit of a stretch to me. It's a pretty simple change though so if you feel strongly about it I can make it so.

Reply:
> I see what you're thinking but It seems like a bit of a stretch to me. It's a pretty simple change though so if you feel strongly about it I can make it so.

I feel strongly enough about it; it seems like a perfect semantic fit to me. That said the discussion about changing the DOM to better match the grid mockup may cause issues with this idea so lets hold off on it for now

Reply:
> > I see what you're thinking but It seems like a bit of a stretch to me. It's a pretty simple change though so if you feel strongly about it I can make it so.
> 
> I feel strongly enough about it; it seems like a perfect semantic fit to me. That said the discussion about changing the DOM to better match the grid mockup may cause issues with this idea so lets hold off on it for now

Done here: https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/443c0e20143d2e2638abcac16552510b4a8bc021